tests/unit/ReferenceListTest.php

```diff
@@ -147,14 +133,6 @@ public function testRemoveReference( ReferenceList $array ) {
 			$this->assertFalse( $array->hasReference( $element ) );
 			$this->assertEquals( --$elementCount, count( $array ) );
 		}
-
-		$elements = $this->getElementInstances();
-		$element = array_shift( $elements );
-
-		$array->removeReference( $element );
-		$array->removeReference( $element );
-
-		$this->assertTrue( true );
```

Comment:
Now the test will not make any assertions for empty lists. This test design is not nice, though now you will get warnings in strict mode if I'm not mistaken